qutebrowser/qutebrowser.py

```diff
@@ -93,6 +93,11 @@ def get_argparser():
 
     parser.add_argument('--json-args', help=argparse.SUPPRESS)
     parser.add_argument('--temp-basedir-restarted', help=argparse.SUPPRESS)
+    parser.add_argument('--app-id',
```

Comment:
Please move this to the block above, the two arguments here are "internal" ones.

Also, unless I'm missing something, this switch (setting Qt's `DesktopFileName`) and Chromium's `--app-id` (taking a Chromium extension ID) are unrelated though?

I'd prefer a name like `--wayland-app-id` or `--desktop-file-name` to avoid confusion.